cookbooks/aws-parallelcluster-environment/resources/efs/partial/_mount_umount.rb

```diff
@@ -101,6 +101,14 @@
     # Path needs to be fully qualified, for example "shared/temp" becomes "/shared/temp"
     efs_shared_dir = "/#{efs_shared_dir}" unless efs_shared_dir.start_with?('/')
     # Unmount EFS
+    Chef::Log.info("Unmounting #{efs_shared_dir} with the following")
+    execute "active processes" do
```

Comment:
What about capturing this block in the spec test?

Reply:
done!